README.md

```diff
@@ -1,7 +1,7 @@
 # espflash
 
-![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/esp-rs/espflash/ci.yml?branch=main&labelColor=1C2C2E&logo=github&style=flat-square)
-![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)
+[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/esp-rs/espflash/ci.yml?branch=main&labelColor=1C2C2E&logo=github&style=flat-square)](https://github.com/esp-rs/espflash/actions/workflows/ci.yml)
+[![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)](https://crates.io/crates/espflash)
```

Comment:
Not sure if this should point to `espflash` on crates.io, as its the readme which contains info about `espflash` and `cargo-espflash`. Tbh, I am not sure if we should even have this badge at all, so Id rather keep it as it was. For the CI badge, I think adding the link is definitely an improvement!

Reply:
```suggestion
![Crates.io](https://img.shields.io/crates/l/espflash?labelColor=1C2C2E&style=flat-square)
```